pkg/apis/kudo/v1beta1/instance_types_helpers.go

```diff
@@ -78,6 +81,9 @@ func (i *Instance) UpdateInstanceStatus(ps *PlanStatus, updatedTimestamp *metav1
 			i.Spec.PlanExecution.Status = ps.Status
 		}
 	}
+	if i.Spec.PlanExecution.Status.IsTerminal() {
```

Comment:
Shouldn't this use `IsFinished` instead? `IsTerminal` includes fatal errors, a plan shouldn't be considered ready for that case.

```suggestion
	if i.Spec.PlanExecution.Status.IsFinished() {
```

Reply:
yes, thank you :)